src/store/modules/product/actions.ts

```diff
@@ -0,0 +1,41 @@
+import { ProductService } from '@/services/ProductService'
+import { ActionTree } from 'vuex'
+import RootState from '@/store/RootState'
+import ProductState from './ProductState'
+import * as types from './mutation-types'
+import { hasError, showToast } from '@/utils'
+import { translate } from '@/i18n'
+import emitter from '@/event-bus'
+
+const actions: ActionTree<ProductState, RootState> = {
+
+  /**
+   * Set Current Product
+   */
+  async setCurrentProduct ( {commit, state}, { productIds }) {
```

Comment:
I think we are fetching the products information to be used on shipment page, we should follow the same pattern as done in preorder
https://github.com/hotwax/preorder/blob/main/src/store/modules/product/actions.ts#L16
Setting current means information set for the current product page

Reply:
Yes Sir, Followed the same pattern.
Thank you sir for reviewing my code.